elastic/kibana
# [Security Solution] Add guided tour to document details flyout

## Summary

This PR adds a guided tour for the new expandable flyout in alerts table. 

Where it will show up:
✅ Alerts page
✅ Alerts tab in Cases

Tour will not show up in:
❌ rule creation
❌ flyout in timeline
❌ event table.

https://github.com/elastic/kibana/assets/18648970/e9d0ce92-0eb4-4898-ad05-91d701aec01d



**How to test**
- Generate some alerts and go to Alerts page
- Expand a row in alerts table
- Guided tour should appear
- Note that rule preview is only available to alerts. Guided tour for an event or alert preview does not have that step.

To test guided tour in event and timeline, enable `expandableEventFlyoutEnabled`, `expandableTimelineFlyoutEnabled` respectively.

https://github.com/elastic/security-team/issues/8134

### Checklist
